doc/sphinx/Pacemaker_Administration/pcs-crmsh.rst

```diff
@@ -0,0 +1,437 @@
+Quick Comparison of pcs and crm shell
+-------------------------------------
+
+``pcs`` and ``crm shell`` are two popular higher-level command-line interfaces
+to Pacemaker. Each has its own syntax; this chapter gives a quick comparion of
+how to accomplish the same tasks using either one. Some examples also show the
+equivalent command using low-level Pacmaker command-line tools.
+
+These examples show the simplest syntax; see the respective man pages for all
+possible options.
+
+Show Cluster Configuration and Status
+#####################################
+
+.. topic:: Show Configuration (Raw XML)
+
+   .. code-block:: none
+
+      crmsh     # crm configure show xml
+      pcs       # pcs cluster cib
+      pacemaker # cibadmin -Q
+
+.. topic:: Show Configuration (Human-friendly)
+
+   .. code-block:: none
+
+      crmsh # crm configure show
+      pcs   # pcs config
+    
+.. topic:: Show Cluster Status
+
+   .. code-block:: none
+
+      crmsh     # crm status
+      pcs       # pcs status
+      pacemaker # crm_mon -1
```

Comment:
It could be worth mentioning `crm_mon --output-as=xml` here, given the previous blocks about showing configuration in multiple formats.

Reply:
Not really worth it, the tools chapter goes into detail about crm_mon. This is primarily about pcs/crmsh; the pacemaker equivalents are sometimes shown just to illustrate what they're doing.